localstack-core/localstack/openapi.yaml

```diff
@@ -580,7 +596,9 @@ paths:
       responses:
         '200':
           content:
-            text/xml: {}
+            text/xml:
+              schema:
+                $ref: '#/components/schemas/ReceiveMessageResult'
```

Comment:
This relates to SQS, not SES :stuck_out_tongue: I was puzzled and spent a while trying to investigate why SES would return XML